analyzer/analyzer.go

```diff
@@ -16,23 +16,41 @@ import (
 	"golang.org/x/tools/go/analysis"
 )
 
+type optionInt struct {
+	enabled bool
+	intConv bool
+}
+
+type optionErr struct {
+	enabled  bool
+	errError bool
+	errorf   bool
+}
+
+type optionStr struct {
+	enabled   bool
+	sprintf1  bool
+	strconcat bool
+}
```

Comment:
Following @mmorel-35 suggestion, I would use another wording, especially because I think using the opposite flag would make code simple.

See https://github.com/catenacyber/perfsprint/pull/41#discussion_r1935366861

Also, I tend to prefer using activated/deactivated than enabled/disabled by respect for people with disabilities.


```suggestion
type optionInt struct {
	deactivated bool
	intConv     bool
}

type optionErr struct {
	deactivated bool
	errError    bool
	errorf      bool
}

type optionStr struct {
	deactivated bool
	sprintf1    bool
	strconcat   bool
}
```

Reply:
Ok for the rewording for the opposite